src/webapp/components/ClusterMapLayers.jsx

```diff
@@ -132,8 +124,12 @@ const ClusterMapLayers = ({ data, paramRequest, sourceId, color }) => {
             sourceId={sourceId}
             filter={["!", ["has", "point_count"]]}
             paint={{
-              "circle-color": color,
-              "circle-radius": 6,
+              "circle-color": ["step",
```

Comment:
If we made the change I suggested above, this would be something like `"circle-color": ["get", "markerColor"]`